### PR TITLE
Add localization for UI strings in Indonesian

### PR DESCRIPTION
**Pull request by stepankuzmin**
Hey 👋

This PR adds translations for UI strings we use in GL JS into Indonesian. We translated the UI strings automatically using the Google Translate API, and we need to verify that the translation is correct.

To help us verify the translations, you can open the `Files changed` tab on this page and compare the translations with the original file [`src/ui/default_locale.js`](https://github.com/mapbox/mapbox-gl-js/blob/main/src/ui/default_locale.js).

To improve the machine translation, click the plus sign next to the line you want to enhance in the `Files changed` tab and suggest a translation.

If the translation looks good, you can approve it in the PR's comments section below.

Thanks for helping 🙌

| Context | Screenshot |
| :------ | :--------: |
| **ID**: `AttributionControl.ToggleAttribution`<br/>**English**: `Toggle attribution`<br/>**Context**: Tooltip text on `AttributionControl` hover | Screenshot 2022-04-12 at 12 22 53
| **ID**: `AttributionControl.MapFeedback`<br/>**English**: `Map feedback`<br/>**Context**: Accessibility label for the `Improve this map` link in the expanded `AttributionControl`. That label can be read by assistive technology | Screenshot 2022-04-11 at 21 40 33
| **ID**: `FullscreenControl.Enter`<br/>**English**: `Enter fullscreen`<br/>**Context**: Tooltip text on FullscreenControl hover when not in the fullscreen mode | Screenshot 2022-04-12 at 12 31 49
| **ID**: `FullscreenControl.Exit`<br/>**English**: `Exit fullscreen`<br/>**Context**: Tooltip text on `FullscreenControl` hover when in the fullscreen mode | Screenshot 2022-04-12 at 12 31 58
| **ID**: `GeolocateControl.FindMyLocation`<br/>**English**: `Find my location`<br/>**Context**: Tooltip text on `GeolocateControl` hover | Screenshot 2022-04-12 at 12 34 57
| **ID**: `GeolocateControl.LocationNotAvailable`<br/>**English**: `Location not available`<br/>**Context**: Tooltip text on `GeolocateControl` hover when we can't locate user | Screenshot 2022-04-12 at 12 35 08
| **ID**: `LogoControl.Title`<br/>**English**: `Mapbox logo`<br/>**Context**: Accessibility label for the Mapbox logo image. That label can be read by assistive technology | Screenshot 2022-04-12 at 12 38 52
| **ID**: `Map.Title`<br/>**English**: `Map`<br/>**Context**: Accessibility label for the map canvas element on the webpage. That label can be read by assistive technology | Screenshot 2022-04-12 at 12 42 03
| **ID**: `NavigationControl.ResetBearing`<br/>**English**: `Reset bearing to north`<br/>**Context**: Tooltip text on bearing button in `NavigationControl` | Screenshot 2022-04-12 at 12 45 06
| **ID**: `NavigationControl.ZoomIn`<br/>**English**: `Zoom in`<br/>**Context**: Tooltip text on plus button in `NavigationControl` | Screenshot 2022-04-12 at 12 45 56
| **ID**: `NavigationControl.ZoomOut`<br/>**English**: `Zoom out`<br/>**Context**: Tooltip text on minus button in `NavigationControl` | Screenshot 2022-04-12 at 12 46 02
| **ID**: `ScrollZoomBlocker.CtrlMessage`<br/>**English**: `Use ctrl + scroll to zoom the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | Screenshot 2022-04-12 at 13 08 03
| **ID**: `ScrollZoomBlocker.CmdMessage`<br/>**English**: `Use ⌘ + scroll to zoom the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | Screenshot 2022-04-12 at 13 08 03
| **ID**: `TouchPanBlocker.Message`<br/>**English**: `Use two fingers to move the map`<br/>**Context**: Text on map blocker alert when cooperative gestures are enabled | IMG_A42AFAB7325C-1

### CODE REVIEW

File: src/ui/locales/id.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
+    "AttributionControl.MapFeedback": "Umpan balik peta",
+    "FullscreenControl.Enter": "Memasuki layar penuh",
```
Comment (christianwbsn) [inline src/ui/locales/id.js:6]: Minor, but for consistency 
```suggestion
    "FullscreenControl.Enter": "Masuk ke layar penuh",
```

File: src/ui/locales/id.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "FullscreenControl.Exit": "Keluar dari layar penuh",
+    "GeolocateControl.FindMyLocation": "Temukan lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "logo kotak peta",
```
Comment (christianwbsn) [inline src/ui/locales/id.js:10]: I assume that Mapbox is a brand name so it doesn't have to be translated
```suggestion
    "LogoControl.Title": "logo Mapbox",
```

File: src/ui/locales/id.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Setel ulang bantalan ke utara",
```
Comment (christianwbsn) [inline src/ui/locales/id.js:12]: "bearing" in the map is a domain-specific word that doesn't have a literal translation, I think we can keep it as "bearing"
```suggestion
    "NavigationControl.ResetBearing": "Setel ulang bearing ke utara",
```

File: src/ui/locales/id.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "NavigationControl.ZoomIn": "Perbesar",
+    "NavigationControl.ZoomOut": "Perkecil",
+    "ScrollZoomBlocker.CtrlMessage": "Gunakan ctrl + gulir untuk memperbesar peta",
+    "ScrollZoomBlocker.CmdMessage": "Gunakan + gulir untuk memperbesar peta",
+    "TouchPanBlocker.Message": "Gunakan dua jari untuk memindahkan peta"
```
Comment (christianwbsn) [inline src/ui/locales/id.js:17]:
```suggestion
    "TouchPanBlocker.Message": "Gunakan dua jari untuk menggeser peta"
```

**Review by christianwbsn — COMMENTED** [review]
Minor updates, mostly for consistency and specific terms that don't have to be translated

File: src/ui/locales/id.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Alihkan atribusi",
```
Comment (minons1) [inline src/ui/locales/id.js:4]: I assume toggle is a button to show or dismiss an information about attribution, so I think it is better translated as show (Tampilkan) in indonesia
```suggestion
    "AttributionControl.ToggleAttribution": "Tampikan atribusi",
```

File: src/ui/locales/id.js (hunk 3 above)
Comment (ardiantovn) [inline src/ui/locales/id.js:12]: Bearing synonym is azimuth [(wikipedia)](https://en.wikipedia.org/wiki/Bearing_(angle)). The translation of azimuth in Indonesian is `azimut`.
```suggestion
    "NavigationControl.ResetBearing": "Setel ulang azimut ke utara",
```

Comment (stepankuzmin) [reply src/ui/locales/id.js]: Do you think it's Okay to use `azimut` as a bearing synonym https://github.com/mapbox/mapbox-gl-js/pull/11702#discussion_r1031062282

**Comment by stepankuzmin** [comment]
Thanks for helping, @christianwbsn, @minons1, and @ardiantovn!

**Comment by minons1** [comment]
Im glad for helping 😄

File: src/ui/locales/id.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// @flow
+
+const locale = {
+    "AttributionControl.ToggleAttribution": "Tampikan atribusi",
```
Comment (PezekZ) [inline src/ui/locales/id.js:4]: Ambil Alih

File: src/ui/locales/id.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    "AttributionControl.MapFeedback": "Umpan balik peta",
```
Comment (PezekZ) [inline src/ui/locales/id.js:5]: Beri Masukan

File: src/ui/locales/id.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    "FullscreenControl.Enter": "Masuk ke layar penuh",
+    "FullscreenControl.Exit": "Keluar dari layar penuh",
+    "GeolocateControl.FindMyLocation": "Temukan lokasi saya",
+    "GeolocateControl.LocationNotAvailable": "Lokasi tidak tersedia",
+    "LogoControl.Title": "logo Mapbox",
+    "Map.Title": "Peta",
+    "NavigationControl.ResetBearing": "Setel ulang azimut ke utara",
```
Comment (PezekZ) [inline src/ui/locales/id.js:12]: Setel Ulang Sudut ke Utara

File: src/ui/locales/id.js [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    "NavigationControl.ZoomIn": "Perbesar",
+    "NavigationControl.ZoomOut": "Perkecil",
+    "ScrollZoomBlocker.CtrlMessage": "Gunakan ctrl + gulir untuk memperbesar peta",
```
Comment (PezekZ) [inline src/ui/locales/id.js:15]: Gunakan Ctrl + Scroll Untuk Memperbesar Peta

File: src/ui/locales/id.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+    "ScrollZoomBlocker.CmdMessage": "Gunakan + gulir untuk memperbesar peta",
```
Comment (PezekZ) [inline src/ui/locales/id.js:16]: Gunakan Ctrl + Scroll Untuk Memperbesar Peta

**Review by PezekZ — COMMENTED** [review]
Hello, i found it in Twitter a week ago, and i'm here to help reviewing this translation and make it easy to understand for Indonesian people

**Pull request merged**